bindings/core/src/method/client.rs

```diff
@@ -282,17 +285,17 @@ pub enum ClientMethod {
     /// Look up a commitment by a given commitment index.
     GetCommitmentByIndex {
         /// Index of the commitment to look up.
-        index: SlotIndex,
+        slot: SlotIndex,
```

Comment:
TIP not updated yet?

Reply:
No, commented already https://github.com/iotaledger/tips/pull/153/files#r1445850201